packages/settings-plugins/settings-plugins.spec

```diff
@@ -72,6 +72,8 @@ Provides: %{_cross_os}settings-plugin(aws-k8s-1.29)
 Provides: %{_cross_os}settings-plugin(aws-k8s-1.29-nvidia)
 Provides: %{_cross_os}settings-plugin(aws-k8s-1.30)
 Provides: %{_cross_os}settings-plugin(aws-k8s-1.30-nvidia)
+Provides: %{_cross_os}settings-plugin(aws-k8s-1.31)
+Provides: %{_cross_os}settings-plugin(aws-k8s-1.31-nvidia)
```

Comment:
Not a blocker, but this will complicate https://github.com/bottlerocket-os/bottlerocket/pull/4125